packages/cms/src/components/sections/Hero.jsx

```diff
@@ -75,6 +77,26 @@ class Hero extends Component {
     }, 300);
   }
 
+  saveToPDF() {
+    const {saving} = this.state;
+    const {router} = this.context;
+    if (!saving) {
+      const url = "/api/pdf";
+      const path = `${router.location.pathname}?print=true`;
+      const payload = {path};
+      const config = {responseType: "arraybuffer", headers: {Accept: "application/pdf"}};
+      this.setState({saving: true});
+      axios.post(url, payload, config).then(resp => {
+        const blob = new Blob([resp.data], {type: "application/pdf"});
+        const link = document.createElement("a");
+        link.href = window.URL.createObjectURL(blob);
+        link.download = "your-file-name.pdf";
```

Comment:
Can you grab the Hero `title` and run it through this `filename` function from `Options.jsx`?

https://github.com/Datawheel/canon/blob/master/packages/cms/src/components/Viz/Options.jsx#L33-L35